src/notation/internal/notationuiactions.cpp

```diff
@@ -180,16 +180,21 @@ const UiActionList NotationUiActions::m_actions = {
              ),
     UiAction("cut",
              mu::context::UiCtxNotationFocused,
-             QT_TRANSLATE_NOOP("action", "Cut")
+             QT_TRANSLATE_NOOP("action", "Cut"),
+             QT_TRANSLATE_NOOP("action", "Cut"),
+             IconCode::Code::CUT
```

Comment:
There is a constructor that allows us to add an icon without having to add a description
Do we really need descriptions here?